tests/e2e/spec/_onboarding.spec.js

```diff
@@ -14,32 +14,32 @@ import {
   getExtensionElement,
   getExtensionPageURL,
   switchToPanel,
+  waitForIdleBackgroundTasks,
 } from '../utils.js';
 
 describe('Onboarding', function () {
-  beforeEach(async function () {
-    await browser.url(await getExtensionPageURL('onboarding'));
-  });
-
-  afterEach(async function () {
-    await browser.url('about:blank');
-  });
-
   it('keeps ghostery disabled', async function () {
+    await browser.url(await getExtensionPageURL('onboarding'));
     await getExtensionElement('button:skip').click();
     await expect(getExtensionElement('view:skip')).toBeDisplayed();
 
     await switchToPanel(async function () {
       await expect(getExtensionElement('button:enable')).toBeDisplayed();
     });
+
+    await browser.url('about:blank');
```

Comment:
why not in afterEach? is that helping with Firefox? if so, maybe add some readme for the tests to share that knowledge

Reply:
Yes, but not only for Firefox. The tests are set for 2 retries, but the before/after hooks do not retry. It is best to avoid them.
We can create README with tips, but I would do that after the test configuration and setup settle down.